lnd_methods/invoices/create_hodl_invoice.js

```diff
@@ -118,6 +119,7 @@ module.exports = (args, cbk) => {
           private: !!args.is_including_private_channels,
           value: args.tokens || undefined,
           value_msat: args.mtokens || undefined,
+          description_hash: description_hash || undefined,
```

Comment:
I think this is missing args, and also the description hash is a buffer so it would need to be converted

Something like

```node
const hexAsBuffer = hex => !!hex ? Buffer.from(hex, 'hex') : undefined;
```

And then

```
description_hash: hexAsBuffer(args.description_hash),
```

Reply:
My bad! Should of course be be `args.description_hash`.

> I think this is missing args, and also the description hash is a buffer so it would need to be converted

Oh yes, right now this code expects `args.description_hash` to be a Buffer.
I'll fix so it's supposed to be a hex-string instead.

Reply:
Fixed.